pick.lic

```diff
@@ -708,48 +714,104 @@ class Pick
 
   def loot(box_noun)
     echo "Looting #{box_noun}..." if @debug
-    if DRC.bput("open my #{box_noun}", /You open/, /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
+    
+    if DRC.bput("open my #{box_noun}", /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
```

Comment:
💡 Could `DRCI.open_container?(box_noun)` work here, and if returns `false` then bail?

Reply:
Implemented as suggested, thankyou